scripts/exregional_make_ics.sh

```diff
@@ -638,20 +621,23 @@ Please check the following user defined variables:
   FVCOM_FILE= \"${FVCOM_FILE}\" "
   fi
 
-  cp_vrfy ${fvcom_data_fp} ${ics_dir}/fvcom.nc
-  cd_vrfy ${ics_dir}
-  ${RUN_CMD_UTILS} ${fvcom_exec_fn} sfc_data.tile${TILE_RGNL}.halo${NH0}.nc fvcom.nc ${FVCOM_WCSTART} ${fvcom_time}|| \
-  print_err_msg_exit "\
+  cp_vrfy ${fvcom_data_fp} ${INPUT_DATA}/fvcom.nc
+  cd_vrfy ${INPUT_DATA}
+  PREP_STEP
+  eval ${RUN_CMD_UTILS} ${fvcom_exec_fn} \
+       ${NET}.${cycle}${dot_ensmem}.sfc_data.tile${TILE_RGNL}.halo${NH0}.nc fvcom.nc ${FVCOM_WCSTART} ${fvcom_time} \
+       ${REDIRECT_OUT_ERR} || print_err_msg_exit "\
```

Comment:
`REDIRECT_OUT_ERR` needs it. Before it was only `RUN_CMD_UTILS` for either `srun/mpirun` that needed it and was done elswhere. Since we need both to be evaled, I though it is easier to do them together.